app/src/main/java/com/example/peterchu/watplanner/MainActivity.java

```diff
@@ -38,6 +40,9 @@ public void onResponse(Call<CourseResponse>call, Response<CourseResponse> respon
                 Log.d("MainActivity", response.toString());
                 List<Course> courses = response.body().getData();
                 Log.d("MyActivity", "Number of courses received: " + courses.size());
+                ListView userCoursesList = (ListView) findViewById(R.id.userCoursesList);
+                userCoursesList.setAdapter(new CourseListAdapter(MainActivity.this,
+                        R.layout.course_list_item_view, courses));
```

Comment:
yup, it's just placeholder data for now - we can just pass in a list of courses to the adapter later once we have the user courses saved.